### PR TITLE
Home Assistant Presence Detection with monitor.sh 

### PR DESCRIPTION
**Pull request by Bas-Man**
This is my personal documentation on how I implemented presence detection using monitor.sh with some additional Raspberry Pis. It contains several reference links that I based things off.

### CODE REVIEW

**Comment by Bas-Man** [comment]
This is complete and ready for review.

File: docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+#### These are links to a single HA user. The config is far beyond what I need. But it's a great resource.
+- [Presence detection - are we nearly there yet?](https://blog.ceard.tech/2019/03/presence-detection-are-we-nearly-there.html)
+- [Presence detection, the final countdown?](https://blog.ceard.tech/2019/10/presence-detection-final-countdown.html)
+- [Presence detection updated](https://blog.ceard.tech/2018/09/a-while-back-i-covered-how-i-was-doing.html)
+- [GitHub Repo](https://github.com/DubhAd/Home-AssistantConfig)
+
+#### Some missing Magic.
+- [The missing magic to make MQTT State usable as a device_tracker](https://community.home-assistant.io/t/combining-multiple-device-trackers-into-one-using-mqtt/45324)
+
+### Software:
```
Comment (IronicBadger) [inline docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md:429]: Please remove `:` as this looks odd in the TOC on the right hand side

File: docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+
+- [[monitor] Reliable, Multi-User, Distributed Bluetooth Occupancy/Presence Detection](https://community.home-assistant.io/t/monitor-reliable-multi-user-distributed-bluetooth-occupancy-presence-detection/68505)
+
+#### These are links to a single HA user. The config is far beyond what I need. But it's a great resource.
+- [Presence detection - are we nearly there yet?](https://blog.ceard.tech/2019/03/presence-detection-are-we-nearly-there.html)
+- [Presence detection, the final countdown?](https://blog.ceard.tech/2019/10/presence-detection-final-countdown.html)
+- [Presence detection updated](https://blog.ceard.tech/2018/09/a-while-back-i-covered-how-i-was-doing.html)
+- [GitHub Repo](https://github.com/DubhAd/Home-AssistantConfig)
+
+#### Some missing Magic.
```
Comment (IronicBadger) [inline docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md:426]: Remove `.`

**Comment by IronicBadger** [comment]
Once review comments about punctuation have been addressed this looks good - great contribution thank you!

File: docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md (hunk 2 above)
Comment (Bas-Man) [reply docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md]: Removed.

File: docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md (hunk 1 above)
Comment (Bas-Man) [reply docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md]: Removed

**Comment by IronicBadger** [comment]
Really great article! 👍 

Thanks for contributing.

**Pull request merged**